docs/_config.yml

```diff
@@ -348,4 +348,12 @@ ja:
       sections:
         - path: ja/week07/07-1.md
         - path: ja/week07/07-2.md
-        - path: ja/week07/07-3.md
\ No newline at end of file
+        - path: ja/week07/07-3.md
+
+################################### Arabic ####################################
+ar:
+  title: 'التعلم العميق'
+  chapters:
+    - path: ar/week01/01.md
```

Comment:
I don't see this file. Is it missing?

Reply:
I removed this line as I didn't translate that file.